src/Core/Vipr/Bootstrapper.cs

```diff
@@ -1,19 +1,19 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
 using System.IO;
-using DocoptNet;
-using Newtonsoft.Json;
-using Vipr.Core;
+using DocoptNet;
+using Newtonsoft.Json;
+using Vipr.Core;
 using Vipr.Core.CodeModel;
 
-namespace Vipr
-{
-    internal class Bootstrapper
-    {
-        const string Usage = @"Vipr CLI Tool
+namespace Vipr
+{
+	internal class Bootstrapper

```

Comment:
Spacing here? Could be tab issue.

Also, I'm concerned that large swaths of files are considered changes when in fact there are no differences. This might be due to line-ending mismatches. Can you verify that we are preserving the line endings?